PLOT/donations.yml

```diff
@@ -0,0 +1,14 @@
+--- 
+- 
```

Comment:
Hi, the CI also requires a USD value (the value of your donation at the time of donating) cann you add that?